app/src/main/java/emu/skyline/AppDialog.kt

```diff
@@ -91,6 +92,12 @@ class AppDialog : BottomSheetDialogFragment() {
             shortcutManager.requestPinShortcut(info.build(), null)
         }
 
+        binding.gameTitleId.setOnLongClickListener {
+            val clipboard = requireActivity().getSystemService(android.content.Context.CLIPBOARD_SERVICE) as android.content.ClipboardManager
+            clipboard.setPrimaryClip(android.content.ClipData.newPlainText("Title ID", item.titleId))
```

Comment:
Show a snack bar after you do this saying "Copied to clipboard".

Reply:
> Show a snack bar after you do this saying "Copied to clipboard".

Done. Also added this line to `strings.xml`